manifests/kustomize/components/monitoring/fluent-bit.yaml

```diff
@@ -51,7 +51,7 @@ valuesInline:
       [INPUT]
           Name   http
           Tag    events
-          Listen 0.0.0.0
+          Listen ::
           Port   9880
```

Comment:
isn't this just ipv6?

Reply:
I believe it supports both. It works locally and we're ipv4 locally. 

```
k get svc -n monitoring
NAME                      TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)             AGE
fluent-bit                ClusterIP   10.43.146.100   <none>        2020/TCP,9880/TCP   4d
grafana                   ClusterIP   10.43.238.163   <none>        80/TCP              4d
victoria-logs-single      ClusterIP   None            <none>        9428/TCP            4d
victoria-metrics-single   ClusterIP   None            <none>        8428/TCP            4d
elasticsearch-master-hl   ClusterIP   None            <none>        9200/TCP,9300/TCP   23h
elasticsearch             ClusterIP   10.43.74.98     <none>        9200/TCP,9300/TCP   23h
elasticsearch-kibana      ClusterIP   10.43.16.106    <none>        5601/TCP            23h
```